cups/md5passwd.c

```diff
@@ -61,35 +50,15 @@ httpMD5Final(const char *nonce,		/* I - Server nonce value */
 	     const char *resource,	/* I - Resource path */
              char       md5[33])	/* IO - MD5 sum */
 {
-  unsigned char		sum[16];	/* Sum data */
-  char			line[1024];	/* Line of data */
-  char			a2[33];		/* Hash of method and resource */
-
-
- /*
-  * First compute the MD5 sum of the method and resource...
-  */
-
-  snprintf(line, sizeof(line), "%s:%s", method, resource);
-  cupsHashData("md5", (unsigned char *)line, strlen(line), sum, sizeof(sum));
-  cupsHashString(sum, sizeof(sum), a2, sizeof(a2));
-
- /*
-  * Then combine A1 (MD5 of username, realm, and password) with the nonce
-  * and A2 (method + resource) values to get the final MD5 sum for the
-  * request...
-  */
-
-  snprintf(line, sizeof(line), "%s:%s:%s", md5, nonce, a2);
-  cupsHashData("md5", (unsigned char *)line, strlen(line), sum, sizeof(sum));
-
-  return ((char *)cupsHashString(sum, sizeof(sum), md5, 33));
+  return NULL;
```

Comment:
See previous comment about stubbing.

Oh, and probably add `(void)arg;` for each of the unused arguments, otherwise I'll be adding them to suppress the "unused arguments" warnings... Thank you!